packages/queue/node-tests/queue-test.js

```diff
@@ -0,0 +1,202 @@
+const {
+  createDefaultEnvironment,
+  destroyDefaultEnvironment
+} = require('@cardstack/test-support/env');
+
+const delay = require('delay');
+
+
+describe('@cardstack/queue', function() {
+  let env, queue;
+
+  async function setup() {
+    env = await createDefaultEnvironment(`${__dirname}/..`);
+
+    queue = env.lookup('hub:queues');
+
+    await env.lookup('hub:indexers').update({ forceRefresh: true });
+  }
+
+  async function teardown() {
+    await destroyDefaultEnvironment(env);
+    await queue.stop();
```

Comment:
This should probably happen automatically when the container is torn down. [See here](https://github.com/cardstack/cardstack/blob/aa4454368343218ff19029f5fb1c3f229b62198c/packages/hub/indexers.js#L89) for an example of using the teardown hook.

That will mean that `destroyDefaultEnvironment` automatically stops the queue.